MessengerOnFileForLinux/CMakeLists.txt

```diff
@@ -70,20 +70,17 @@ add_subdirectory(${MODULE_TerminalFunctionality})
 add_subdirectory(${MODULE_Common})
 add_subdirectory(${MODULE_DisplayChat})
 add_subdirectory(${MODULE_MessageFramework})
-# CMAKE BUILD
-add_executable(messenger_binary ${SOURCES} ${MAIN})
-target_link_libraries (messenger_binary ${CMAKE_THREAD_LIBS_INIT})
-target_link_libraries(messenger_binary ${CURSES_LIBRARIES})
 
 ################################################################################
 #####                       MAKE UNIT TESTS CMAKE PART                     #####
 ################################################################################
-#LIB_TEST
+# LIB_TEST
 include_directories(${CMAKE_SOURCE_DIR}/ChatStarter/test)
 include_directories(${CMAKE_SOURCE_DIR}/UserService/test)
 include_directories(${CMAKE_SOURCE_DIR}/MessageFramework/test)
 include_directories(${CMAKE_SOURCE_DIR}/TerminalFunctionality/test)
-# TEST
+include_directories(${CMAKE_SOURCE_DIR}/DisplayChat/test)
+#TEST
```

Comment:
raz dodałeś spację, a tutaj bez

Reply:
done

Reply:
Przypominam, że to nie CPLANE, więc jak robisz poprawkę to ją wcommituj tutaj, a nie mów tylko, że jest zrobiona.